### ISSUE TITLE
How to view docker secrets in a Dockerfile

### ISSUE THREAD

**Issue by AmirulAndalib** [wagoodman/dive#411]
Hi 

![Screenshot_20220721-101930](https://user-images.githubusercontent.com/86344856/180129168-805ad562-8165-496c-aad8-65d2efe9242f.png)

Is there any way to view this docker secret?

**Comment by rufanov** [comment]
Secrets not stored inside container. It's stored on host - you should set them yourself to desired values before running container.

**Comment by AmirulAndalib** [comment]
> Secrets not stored inside container. It's stored on host - you should set them yourself to desired values before running container.

It's. Not mine but I wanted to explore it out

**Comment by rufanov** [comment]
@AmirulAndalib , no, they don't. Docker/Podman/Kubernetes just mounts secret data to `/run/secrets/<secret_name>` file's, so app inside container can read it at runtime.

This feature made exactly as simple way  to manage secret data(passwords, private keys, etc) that should never be stored inside container.
